default.nix

```diff
@@ -39,6 +39,11 @@
 with pkgs;
 with stdenv.lib;
 
+let packages = [ "coq" ]
+++ optionals buildIde [ "coqide" "coqide-server" ]
```

Comment:
You can always build `coqide-server` if you want, it doesn't include any gui library, only a different toplevel.

Reply:
Thanks for the tip.

Reply:
In particular it is used for the test-suite so indeed it may be useful to have around.